docusaurus.config.js

```diff
@@ -130,6 +119,35 @@ const config = {
           crossorigin: 'anonymous',
         },
       ],
+      algolia: {
+        // The application ID provided by Algolia
+        appId: '97APAVUL6H',
+
+        // Public API key: it is safe to commit it
+        apiKey: '8cdbbd8b0b98caab7765766267b1cb23',
+
+        indexName: 'noir-lang',
+
+        // Optional: see doc section below
+        contextualSearch: true,
+
+        // Optional: Specify domains where the navigation should occur through window.location instead on history.push. Useful when our Algolia config crawls multiple documentation sites and we want to navigate with window.location.href to them.
+        externalUrlRegex: 'external\\.com|domain\\.com',
```

Comment:
I think this was supposed to be `noir-lang.org`, right?

Reply:
Oh, this is actually supposed to be external sites that we link out to that algolia might crawl. So probably just fine to remove it